benchmark/solver/solver.cpp

```diff
@@ -81,8 +81,7 @@ DEFINE_bool(overhead, false,
 
 
 // input validation
-[[noreturn]] void print_config_error_and_exit()
-{
+[[noreturn]] void print_config_error_and_exit() {
```

Comment:
We should do something about this formatting flip-flopping

Reply:
Is it due to the combination of the `[[noreturn]]` and the `clang-format` version ?

Reply:
Likely yes, this formatting has been changed a few times in the last months, see git blame ;)

Reply:
@upsj Do you use clang-format 6?

Reply:
yes